.github/workflows/ci.yml

```
@@ -163,3 +163,23 @@ jobs:
       with:
         name: ${{ matrix.browser-channel }}-${{ matrix.os }}
         path: pw-log.txt
+
+  build-conda:
+    name: Conda Build
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
```

I would prefer there using the exact versions, sometimes they change it and then its's e.g. broken.
```suggestion
        os: [ubuntu-20.04, macos-10.15, windows-2019]
```

Updating and bumping them manually every 2 years is preferred.